packages/teams-js/src/public/runtime.ts

```diff
@@ -171,16 +171,23 @@ export const versionConstants: Record<string, Array<ICapabilityReqs>> = {
       capability: { teams: { fullTrust: { joinedTeams: {} } } },
       hostClientTypes: [
         HostClientType.android,
+        HostClientType.desktop,
+        HostClientType.ios,
         HostClientType.teamsRoomsAndroid,
         HostClientType.teamsPhones,
         HostClientType.teamsDisplays,
+        HostClientType.web,
       ],
     },
+    {
+      capability: { webStorage: {} },
+      hostClientTypes: [HostClientType.desktop, HostClientType.web],
```

Comment:
If you only add `hostClientTypes: [HostClientType.desktop],` here, and make the change in webStorage.ts, then I confirmed that the webStorage.spec.ts tests pass without any changes.

Because of that, I suggest @reenudeswal reviews this to see if this change to add `HostClientType.web` as well is something that should occur.

Reply:
Web storage is never cleared on Teams web client so we should not add HostClientType.web in   '2.0.5': [
    {
      capability: { webStorage: {} },
      hostClientTypes: [HostClientType.android, HostClientType.ios],
    },